pkg/azurediskplugin/main.go

```diff
@@ -271,11 +217,13 @@ func getDriverConfig() *azdiskv1beta2.AzDiskDriverConfiguration {
 			}
 		})
 	}
+
 	driverConfig.NodeConfig.NodeID = *nodeID
-	if driverConfig == (azdiskv1beta2.AzDiskDriverConfiguration{}) {
+	if *driverConfig == (azdiskv1beta2.AzDiskDriverConfiguration{}) {
```

Comment:
Since `driverConfig` is a pointer and is initialized conditionally (although currently it will always be initialized), we should probably check for nil before dereferencing and checking for the empty value.